src/client/camera.cpp

```diff
@@ -311,7 +311,7 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime, f32 tool_r
 
 	// Fall bobbing animation
 	float fall_bobbing = 0;
-	if(player->camera_impact >= 1 && m_camera_mode < CAMERA_MODE_THIRD)
+	if(player->camera_impact >= 1 && m_camera_mode == CAMERA_MODE_FIRST)
```

Comment:
```suggestion
	if (player->camera_impact >= 1 && m_camera_mode == CAMERA_MODE_FIRST)
```

Reply:
Dealt with in latest commit (though the formatting here wasn't mine)